pkg/models/iam/am/am.go

```diff
@@ -112,7 +113,13 @@ type amOperator struct {
 	k8sclient                  kubernetes.Interface
 }
 
-func NewReadOnlyOperator(factory informers.InformerFactory) AccessManagementInterface {
+func NewReadOnlyOperator(factory informers.InformerFactory, config *config.Config) AccessManagementInterface {
+	// Make devops project lister initialize when devops.devopsServiceAddress has text
```

Comment:
> Make devops project lister initialize when devops.devopsServiceAddress has text

Make devopsProjectLister initialize when devops.devopsServiceAddress is not empty

And please add a description to tell that the DevOps related code lines will be removed in the future version. For example:

`we keep the DevOps related code lines until DevOps becomes a full independent project`

Reply:
Good suggestion.